<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remed_author</class>
 <widget class="QDialog" name="remed_author">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Информация о препарате</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="remedyFullName">
           <property name="text">
            <string>remed name full</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="keysButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Ключи</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="chapterPath">
         <property name="text">
          <string>chapter path</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="puctureButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Изображение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soundButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Звук</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,20">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget_2</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>remed_author</receiver>
   <slot>showTextInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keysButton</sender>
   <signal>clicked()</signal>
   <receiver>remed_author</receiver>
   <slot>showRemedList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>puctureButton</sender>
   <signal>clicked()</signal>
   <receiver>remed_author</receiver>
   <slot>openPhotoViewer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundButton</sender>
   <signal>clicked()</signal>
   <receiver>remed_author</receiver>
   <slot>openSoundViewer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showTextInformation()</slot>
  <slot>showRemedList()</slot>
  <slot>openPhotoViewer()</slot>
  <slot>openSoundViewer()</slot>
 </slots>
</ui>
